src/main/java/jenkins/plugins/nodejs/tools/pathresolvers/LatestInstallerPathResolver.java

```diff
@@ -105,7 +105,10 @@ public String resolvePathFor(String version, Platform platform, CPU cpu) {
             break;
         case amd64:
             if (platform == Platform.SUNOS && //
-                    (new NodeJSVersionRange("[7, 7.5)").includes(nodeVersion) || nodeVersion.compareTo(new NodeJSVersion(0, 12, 18)) == 0)) {
+                    (   new NodeJSVersionRange("[7, 7.5)").includes(nodeVersion) ||
```

Comment:
Please revert changes not related to this PR

Reply:
Changes reverted (note that that version is not available as download)